### ISSUE TITLE
Popup buttons missing visible focused state indicator on keyboard navigation in Chrome

### ISSUE THREAD

**Issue by ghostwords** [EFForg/privacybadger#3106]
To reproduce:

1. Using Chrome, navigate to some site
2. Click on PB's icon in the browser toolbar. If you don't see PB's icon on the toolbar, look for it inside the "puzzle piece" icon menu.
3. Click inside the popup (related bug: #3105)
4. Start pressing <kbd>Tab</kbd>
5. Observe that the "trackers" link gets a focus highlight
6. Press <kbd>Tab</kbd> again to put focus on the "Disable for this site" button
7. Observe that there is (almost) no visible highlight on the button in Chrome

Affected buttons (`pbButton` class buttons):

- Disable for this site / Enable for this site
- Report Broken Site
- Send report
- Cancel
- Copy to clipboard

We need to make sure the focus highlight is visible in light and dark modes on both Chrome and Firefox.
